packages/boot/tools/ibc/mocks.js

```diff
@@ -1,4 +1,5 @@
 // @ts-check
+import { createMockAckMap } from '@agoric/orchestration/tools/ibc-mocks.ts';
```

Comment:
`.ts` doesn't seem right here. Is the correct way to address this a build step in the orchestration package.json?

Reply:
Just change it to `.js`. TS will resolve the module.
```suggestion
import { createMockAckMap } from '@agoric/orchestration/tools/ibc-mocks.js';
```

I don't recall why boot has `allowImportingTsExtensions: true`. I'll see about disabling it to avoid this problem.

>a build step in the orchestration package.json?

Aside from cosmic-proto which uses code gen, we avoid build steps for dependencies between packages in the repo. `@agoric/orchestration` does transpile in its NPM publishing.

Reply:
After rebasing on #9884 and changing to `.js` I see 🤔 :
```
$ ava test/orchestration/restart-contracts.test.ts

  Uncaught exception in test/orchestration/restart-contracts.test.ts

  Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/0xpatrick/repos/agoric-sdk/node_modules/@agoric/orchestration/tools/ibc-mocks.js' imported from /Users/0xpatrick/repos/agoric-sdk/packages/boot/tools/ibc/mocks.js

  › h (file:///Users/0xpatrick/repos/agoric-sdk/node_modules/@esbuild-kit/esm-loader/dist/index.js:12:1164)
  ```

Reply:
pushed a fix in c4d45bf10562dc7375edddbdff355bd8f4df376b

Reply:
Thanks. Still seeing https://github.com/Agoric/agoric-sdk/actions/runs/10419531180/job/28857889091?pr=9882#step:5:115. The problem goes away if we rename `mocks.js` and `mocks.test.js` to `.ts`, but I'll hold that fixup as I suspect a different solution might be preferred.

Reply:
shoot, I tested locally but missed that one I guess.

more `.ts` seems like the best solution to me

Reply:
this branch is behind master now. when you make that change feel free to rebase on master and force push with fixups squashed